Parse/Internal/ParseManager.m

```diff
@@ -108,38 +108,32 @@ - (instancetype)initWithApplicationId:(NSString *)applicationId clientKey:(NSStr
     _controllerAccessQueue = dispatch_queue_create("com.parse.controller.access", DISPATCH_QUEUE_SERIAL);
     _preloadQueue = dispatch_queue_create("com.parse.preload", DISPATCH_QUEUE_SERIAL);
 
-    _applicationId = [applicationId copy];
-    _clientKey = [clientKey copy];
+    _configuration = [configuration copy];
 
     return self;
 }
 
-- (void)configureWithApplicationGroupIdentifier:(NSString *)applicationGroupIdentifier
-                containingApplicationIdentifier:(NSString *)containingApplicationIdentifier
-                          enabledLocalDataStore:(BOOL)localDataStoreEnabled {
-    _applicationGroupIdentifier = [applicationGroupIdentifier copy];
-    _containingApplicationIdentifier = [containingApplicationIdentifier copy];
+///--------------------------------------
+#pragma mark - Offline Store
```

Comment:
Not sure if this is the appropriate place for it, as `startManaging` doesn't quite relate to Offline Store only.